include/zephyr/pm/device.h

```diff
@@ -667,6 +683,19 @@ static inline bool pm_device_is_powered(const struct device *dev)
 	ARG_UNUSED(dev);
 	return true;
 }
+
+static inline int pm_device_driver_init(const struct device *dev, pm_device_action_cb_t action_cb)
+{
+	int rc;
+
+	/* When power management is not enabled, all drivers should initialise to active state */
+	rc = action_cb(dev, PM_DEVICE_ACTION_TURN_ON);
+	if (rc == 0) {
+		rc = action_cb(dev, PM_DEVICE_ACTION_RESUME);
+	}
+	return rc;
```

Comment:
This may be  confusing, if the target is built without `PM_DEVICE` enabled, we don't have a `struct pm_device` associated with it and consequently no tracking of device power states. My take is that this exists to re-use the action callback and avoid code duplication. Which is a valid reason imho.

Reply:
blahh ... I've jumped into the changes before looking the pr message :)